Lib/test/test_capi/test_misc.py

```diff
@@ -2033,6 +2035,9 @@ def test_configured_settings(self):
                 (THREADS | EXTENSIONS, False),
         }.items():
             kwargs = dict(zip(kwlist, config))
+            if Py_GIL_DISABLED and not kwargs['check_multi_interp_extensions']:
+                # Skip unsupported configuration
+                continue
```

Comment:
Likewise, could we incorporate some sort of "expected failure" logic here, to avoid the problem with skipped tests.  I understand if that would be too much work or too messy.  Either way, it would help to have a comment here referring to gh-117649.  That would at least make this searchable.